packages/ui/src/components/Button/Button.tsx

```diff
@@ -44,15 +44,13 @@ export function Button({
                 disabled:bg-white disabled:border-transparent disabled:text-gray-400
                 `,
                 Boolean(Icon) && Boolean(children) && `gap-2`,
-                Boolean(Icon) && !children && `px-4 py-3`,
                 className,
             )}
             disabled={disabled || isLoading}
             {...buttonProps}
         >
             {iconPlacement === 'left' && <Icon icon={icon} />}
-            {/* If there are no children, we insert a zero-width space to preserve the line-height */}
-            <span className={contentClassName}>{children ?? <>&#8203;</>}</span>
```

Comment:
Removing this fix made the square button smaller than regular button (50px vs 54px).
Please test more carefully when fixing UI bugs 🙏🏻

Reply:
Fixed https://github.com/prezly/theme-kit-js/pull/404/commits/4429bf7ad83307b0cdc2850c7b4d3bc846b2f799